patch/files/scripts/variables.gd

```diff
@@ -17,8 +17,15 @@ var ropewearoutrate = 10.0
 var learnpointsperstat = 3.0
 var attributepointsperupgradepoint = 1.0
 var specializationchance = 5.0
-
-
+#ralphC - Mana Eater Mechanics (incl Succubus, Golem, etc.)
+var basemanafoodconsumption = 10.0 #ralphC
+var orgasmmana = 3 #ralphC - 3 is default
+var succubusagemod = { #ralphC - [child,teen,adult] Succubus base mana food consumption multiplier
+	child = 1.0, teen = 2.0, adult = 3.0
+}
+var succubushungerlevel = [1.5,3.0,7.0] #ralphC - [munchies = 1.5, sex crazed 3, death 7] see mansion.gd
+var succubuscounter = 0 #ralphC - actually just used to count orgasms drained by succubi in newsexsystem.gd
```

Comment:
This file is not an appropriate location for `succubuscounter`, a limited scope value that changes during gameplay. Please move this to newsexsystem.gd.

Reply:
Yeah I know, but it was expedient vs adding another argument (I think the right term?) to pass into the function I need to use since it's called elsewhere ~9 to 12 times with nothing to do with succubi.
Would that be better? It means my commit needs to touch a bunch more unrelated scripts.

Reply:
the member class already has a reference to the newsexsystem.gd file scope so the code structure would be identical to what you are currently using.

Reply:
I tried a couple times and failed to make it work that way.  I know this is kind of like using a saw for a doorstop, but it's minimally invasive atm so I'm leaving it alone.  If you feel like changing it, I'll examine and learn from how you did it, but for now I'm more worried about screwing up newsexsystem.gd by playing with a bunch of function arguments.  I'll come back to this once I've (re)learned how to use functions and classes properly.